app/scripts/copy-db.sh

```diff
@@ -19,7 +19,7 @@ parse_connection_string() {
 
 # Function to check whether to dump the prod DB
 should_dump_prod_db() {
-    if [ ! -f "/tmp/openpipe-prod.db" ] || [ "$FORCE_DUMP" = "true" ]; then
+    if [ ! -f "$PROD_DUMP_FILE" ] || [ "$FORCE_DUMP" = "true" ]; then
```

Comment:
Where is this variable defined? It should either be a required cli arg or have a default value (or both)

Reply:
It's defined in your .env file. Since most developers won't use this script, I'm okay with this DX, but you're welcome to improve it.